### PR TITLE
Recipe App

### PR DESCRIPTION
**Pull request by bushmusi**
# Hi there, We have added below feature

- [x] Testing, Authentication using `devise`, Authorization using ``cancancan`
- [x] A login page.
- [x] A registration page.
- [x] A food list (with all CRUD implementation, except for 'update').
- [x] A recipes list (with all CRUD implementation, except for 'update').
- [x] A general shopping list view (all missing food for all your recipes and total price).
- [x] A list of all public recipes from other users with their names and total prices.

## [Live Demo](https://habesha-recipe.herokuapp.com/)

**We make a seeder file you can use for testing**

### CODE REVIEW

**Review by KDlamini — CHANGES_REQUESTED** [review]
### Hi team @bushmusi, @ZahraArshia, @Beardless-sheik 🙋‍♀️,  

Excellent work so far! 💯 
You have done a great job creating a working Recipe app and implementing all the project requirements.
There are some issues that you still need to work on to improve your project according to the requirements. You are almost there! 💪🏾 

## Highlights 🌞
✔️  Good UI, navigation, and following the wireframe layout
✔️  Excellent implementation of models, migrations, controllers, and views
✔️  Required tests are written and they all pass
✔️  Required gems like Device, Cancancan, Rspec, and Capybara are used
✔️  Good use of branches to introduce new features and perform peer-reviews
✔️  Good PR title and description
✔️  Commit messages are meaningful
✔️  No linter errors

## Required Changes ♻️
 
 - [ ] Please update your README to make it more relevant to this project. 
 - The project title and description is currently for a Blog app, Kindly make it relevant to this project.
     
image

- Under configuration: the database creation name is for the blog_app database, kindly include the database for this project. This section can also be more descriptive about how to set up the database as well.

     For example:
image

- You are missing the `Getting Started` section as well which should explain how to clone and set up the project locally.

    For example:
image

- Under authors: since you are a group of three there are credentials missing for one author @Beardless-sheik 

- On the issues link: kindly add a link for the issues page of the repository of this project. It currently links to the `my-catalogue` repository.

- Kindly make sure you have a working link to your license, this link is currently broken.

image


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

N/A

Cheers 🥂  and Happy coding!!! 👯 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @KDlamini  in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by bushmusi** [comment]
It is really helpful comment @KDlamini. Thanks so much

File: README.md [hunk 1]
```diff
@@ -53,7 +73,3 @@ Give a ⭐️ if you like this project!
 ## Acknowledgments
 
 - Microverse: [microverse community](https://github.com/microverseinc)
-
-## 📝 License
-
-This project is [MIT](./MIT.md) licensed.
```
Comment (Rustamxon7) [inline README.md]: - [ ] [OPTIONAL] All requirements implemented as requested by the previous reviewer. Please create a `MIT.md` license file and add the link you removed last time 👍. You can get the template from [this link](https://github.com/microverseinc/readme-template/blob/master/MIT.md)

**Review by rustam-tolipov — APPROVED** [review]
## Hi team👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 🎉 🎉 🎉 🎉 

Cheers and Happy coding!👏👏👏

Developer-512px-10

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: README.md (hunk 1 above)
Comment (bushmusi) [reply README.md]: Hah thank @Rustamxon7 :+1:

**Pull request merged**